### PR TITLE
[25.x] 2293 add page for upgrade tags (#2301)

### LINKED ISSUE #2417: [Backport]: backport 2293 add page for upgrade tags
### BC Idea Link

https://github.com/microsoft/BCApps/issues/2293

### Description

Backport to BC25.x
@nikolakukrika could you approve as requested?

### I will provide the implementation for this BC Idea

- [x] I will provide the implementation for this BC Idea

### PR DESCRIPTION
**Pull request by Bertverbeek4PS**
#### Summary
It would be very welcome to add a page to the upgrade tags table so in that case you can see what kind of upgrade tags there are executed for troubleshooting.

#### Work Item(s) 
Fixes #2417

Fixes
[AB#556569](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/556569)

---------

#### Work Item(s) 
Fixes #

### CODE REVIEW

**Comment by Bertverbeek4PS** [comment]
@nikolakukrika this is the pull request to backport it.

**Comment by JesperSchulz** [comment]
@Bertverbeek4PS, this is in fact the first ever partner processed backport! Good job 🥳

**Review by Aleyenda — DISMISSED** [review]

**Review by nikolakukrika — DISMISSED** [review]

File: src/System Application/App/Upgrade Tags/src/UpgradeTags.Page.al [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Upgrade;
+page 9985 "Upgrade Tags"
```
Comment (JesperSchulz) [inline src/System Application/App/Upgrade Tags/src/UpgradeTags.Page.al:7]: Not sure why it is complaining about the already existing ID. The page doesn't exist and the ID isn't used. We'll do a swift investigation!

Comment (JesperSchulz) [reply src/System Application/App/Upgrade Tags/src/UpgradeTags.Page.al:7]: No firsts without small hiccups 😝

Comment (JesperSchulz) [reply src/System Application/App/Upgrade Tags/src/UpgradeTags.Page.al:7]: So, it turns out that ID is taken in 25: https://github.com/microsoft/BCApps/blob/releases/25.x/src/System%20Application/App/Word%20Templates/src/WordTemplatesRelatedList.Page.al

Now, the page is deprecated, it isn't used and it's not extensible, so I'd be fine with deleting it. You would have to update your PR with that change.

Comment (Bertverbeek4PS) [reply src/System Application/App/Upgrade Tags/src/UpgradeTags.Page.al:7]: Yeps no problem. Deleted that page 😃

**Review by JesperSchulz — APPROVED** [review]

**Review by nikolakukrika — APPROVED** [review]

**Pull request merged**